src/components/password-input/index.md

```diff
@@ -149,7 +149,7 @@ Some browsers might automatically change what the user is typing when the input
 
 You can tell browsers not to correct spellings by setting the `spellcheck` attribute to `false`.
 
-Doing this can also prevent ['spell-jacking'](https://www.otto-js.com/news/article/chrome-and-edge-enhanced-spellcheck-features-expose-pii-even-your-passwords), where security researchers have found some spell checking tools gathering personal identifiable information, even user’s passwords, from password input fields to send to third party services.
+Doing this can also prevent 'spell-jacking', where security researchers have found some spell checking tools gathering personal identifiable information, even user’s passwords, from password input fields to send to third party services.
```

Comment:
@owenatgov What do you think of this source? Of what I've found it looks like a mostly reputable publication and likely to keep their content up.

Alternatively, I can rewrite this line to just say "prevent spellchecking tools from storing personally identifiable information" so we avoid throwing unsubstantiated claims around.

```suggestion
Doing this can also prevent ['spell-jacking'](https://www.itpro.com/security/vulnerability/370010/what-is-spell-jacking), where security researchers have found some spell checking tools gathering personal identifiable information, even user’s passwords, from password input fields to send to third party services.
```

Reply:
They seem fine to me. I had a look at the article and around the site and it seems fine. Reddit obviously isn't a concrete source but [this thread on itpro](https://www.reddit.com/r/ITCareerQuestions/comments/vievgy/itprotv_good_or_bad_advice_needed/) suggests they're good.

The only downside is that the article references the same otto js article that we're trying to replace 😂 However they also explain it in the text so it's not a huge issue. Folks can read this and understand what spell-jacking is in broad.

I'll commit this for now but tag @querkmachine in case it wants to dispute this as a source.